src/views/Tasks/ViewTask/index.jsx

```diff
@@ -688,7 +691,11 @@ export default class ViewTask extends Component {
         helpView={<HelpView description={description} />}
         search={<Search onSubmit={this.handleTaskSearchSubmit} />}>
         {loading && <Spinner loading />}
-        <ErrorPanel error={error} warning={Boolean(task)} />
+        <ErrorPanel
+          error={error}
+          warning={Boolean(task)}
+          className={classes.errorPanel}
```

Comment:
Same thing here; we only want the class to be active when the `warning` prop is set. I would also probably rename the class name to `warningPanel` instead of `errorPanel`.